four_wheel_steering_controller/test/src/four_wheel_steering.cpp

```diff
@@ -13,19 +14,53 @@ int main(int argc, char **argv)
   ros::init(argc, argv, "four_wheel_steering");
   ros::NodeHandle nh;
 
+  // This should be set in launch files
+  // as well
+  nh.setParam("/use_sim_time", true);
+
   FourWheelSteering robot;
   ROS_WARN_STREAM("period: " << robot.getPeriod().toSec());
   controller_manager::ControllerManager cm(&robot, nh);
 
-  ros::Rate rate(1.0 / robot.getPeriod().toSec());
+  ros::Publisher clock_publisher = nh.advertise<rosgraph_msgs::Clock>("/clock", 1);
```

Comment:
@bmagyar @vincentrou Again, similarly to #318, the simulation publishes its own clock which should fix time/integration erratic failures.